dev-tex/glossaries/glossaries-4.37.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
This copyright mark is old. Upgrade your portage and repoman please, to fix it automatically for you when committing.